terraform/modules/ci/outputs.tf

```diff
@@ -18,9 +18,16 @@ output "server_url" {
   value = module.server_service.audit_log_server_url
 }
 
-output "client_endpoints" {
+output "ingestion_client_endpoints" {
   value = {
     for key, value in var.client_images :
-    key => lookup(google_cloud_run_service.client_services, key).status[0].url
+    key => lookup(google_cloud_run_service.ingestion_client_services, key).status[0].url
+  }
+}
+
+output "cl_client_endpoints" {
+  value = {
+    for key, value in var.client_images :
+    key => lookup(google_cloud_run_service.cl_client_services, key).status[0].url
   }
 }
```

Comment:
Will it be easier in the workflow if we use https://developer.hashicorp.com/terraform/language/functions/merge or https://developer.hashicorp.com/terraform/language/functions/concat to combine the output in `clients_endpoints`?

Reply:
It is possible, but it makes it more difficulty to debug with some specific tests. For example, I just want to debug with java grpc(cloud logging).